src/tools/compiletest/src/main.rs

```diff
@@ -211,6 +211,13 @@ pub fn parse_config(args: Vec<String>) -> Config {
         // Avoid spawning an external command when we know tidy won't be used.
         false
     };
+    let rustc_cfg_data = Command::new("rustc")
```

Comment:
You should use the `rustc-path` from the config below instead of the plain `rustc` that you wrote.

Reply:
Thank you!